packages/apps/graph/src/graph/types/graphql-types.ts

```diff
@@ -123,9 +123,16 @@ export interface TransactionMeta {
 }
 
 // eslint-disable-next-line @typescript-eslint/naming-convention
+export interface MempoolInfo {
+  status: string;
+}
 
-export interface TransactionResult {
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export interface TransactionInfo {
+  hash: string;
+  chainId: bigint;
```

Comment:
I added these two here because I need this data to be able to get `Block`, `Events`, and `Transfers` in the `TransactionInfo` object. However, these are not exposed in the `TransactionInfo` output